scripts/set_new_session_key

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Permission to simplify this script a bit and rename to `author_rotate_keys`?

Reply:
Set new session key seemed more obvious

Reply:
`set_new_session_key` may be a little misleading because setters usually take the item being set as an argument. That's the concern I had.

`author_rotate_keys` mirrors the jsonrpc call that this wraps "author_rotateKeys".

Reply:
Not a blocker btw.

Reply:
Ok

Reply:
Changed